kube/jenkins-deployment.yaml

```diff
@@ -9,14 +9,20 @@ spec:
       labels:
         app: jenkins
     spec:
+      serviceAccountName: jenkins
       containers:
         - name: jenkins
-          image: liatrio/kube-jenkins:latest
+          image: liatrio/kube-jenkins:10
           env:
+            # Read the configuration-as-code from the ConfigMap
+            - name: CASC_JENKINS_CONFIG
+              value: /var/jenkins_config/configuration-as-code.yaml
+            # Load a joblist from a json file
             - name: JOBLIST
-              value: https://raw.githubusercontent.com/liatrio/liatrio-jenkins/master/jenkins/jobs.json
+              value: https://raw.githubusercontent.com/liatrio/liatrio-jenkins/kube_secrets/example/jobs.json
```

Comment:
Should we be pulling values like this?

Reply:
Jobs can be added directly to the configuration as code config in jenkins-config.yaml,  to the original jenkins/jobs.txt list, or passed in via this joblist string. Depending on how we deploy, or whether want to clone this repo over and over, one method might be better than the other. This method allowed me to de-couple the job list from the jenkins repo.